components/Tabs/Tabs.js

```diff
@@ -2,43 +2,63 @@
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
+    this.clElement = this.element.classList;
   }
 
   select() {
     // should use classList
+    this.clElement.add("Tabs__item-selected");
   }
 
   deselect() {
     // should use classList
+    this.clElement.remove("Tabs__item-selected");
   }
 }
 
 class TabLink {
   constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
-    // reassign this.tabItem to be a new instance of TabItem, passing it this.tabItem
+    this.element = element;// attach dom element to object
+    this.tabs = parent;// attach parent to object
+    // assign this to the associated tab using the parent's "getTab" method by passing it the correct data
```

Comment:
Keep comment style consistent some are to the side others are above. This might show inexperience to some reviewers.